### ISSUE TITLE
pFlash bin file?

### ISSUE THREAD

**Issue by nik11s4** [bri3d/Simos18_SBOOT#2]
Can you send me the pFlash bin file from which you got the access protocol in "sboot"? Thanks.
nik11s4@gmail.com

**Comment by scandude1** [comment]
Say what now?  I've never read a simos18 in boot

**Comment by jakka351** [comment]
Yeah what? Me either bro...I've emailed you a Ford Falcon Spanish Oak file hope that helps.
